hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -561,7 +625,7 @@ public void cleanup() {
     }
 
     public static interface Announcer {
-        public void announce(long stateVersion);
+        public void announce(long stateVersion, Map<String, String> metadata);
```

Comment:
This was implemented for the use case to notify client about schema changes on delta.   Any suggestions on an alternative way to propagate this information?